CHANGELOG.md

```diff
@@ -9,18 +9,37 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
-* Telemetry
-* RTT logging
+* Telemetry (#341)
+* Logging via RTT (real time tracing) over SWD/JTAG instead of semihosting
+  for fast and low-overhead debugging (#393 #391 #358)
 * Settings structures now automatically publish after a short delay on boot
+  (#475)
+* Full-rate data streaming (e.g. complete raw ADC/DAC data) via Ethernet/UDP
+  (#414 #394 #380)
+* Process routine can be executed from fast closely coupled memory with wide
```

Comment:
This feels like it should be CHANGED, since it doesn't affect the end user really. Just makes it more efficient.